modin/pandas/test/test_io.py

```diff
@@ -953,6 +993,21 @@ def test_read_csv_names_neq_num_cols(self, request, kwargs):
             **kwargs,
         )
 
+    def test_read_csv_wrong_path(self):
+
+        raising_exceptions = list(io_ops_bad_exc).remove(FileNotFoundError)
+
+        eval_io(
+            fn_name="read_csv",
+            raising_exceptions=raising_exceptions,
+            # read_csv kwargs
+            filepath_or_buffer="/some/wrong/path.csv",
+        )
+
+    @pytest.mark.skipif(
+        Backend.get() == "Omnisci",
+        reason="to_csv is not implemented with OmniSci backend yet",
```

Comment:
maybe submit a tracker and reference it here?

Reply:
Fixed.